python/setup.py

```diff
@@ -235,6 +235,14 @@ def get_packages(self):
         "adag": [
             "cupy-cuda12x; sys_platform != 'darwin'",
         ],
+        "azure": [
+            "azure-cli-core",
```

Comment:
does the version matter here? it used to have version?

Reply:
I don't think the version matter here. It used to have version since it is used in CI. But now it is an extra package for users to install, so version doesn't matter.